README.md

```diff
@@ -42,7 +42,7 @@ epilogue.initialize({
 });
 
 // Create REST resource
-var userResource = epilogue.resource({
+var users = epilogue.resource({
```

Comment:
Feel free to change/modify as you see fit. I went for path of least resistance as I'm not a user of epilogue, just a passing reader.